chart/epinio-installer/questions.yml

```diff
@@ -52,12 +52,12 @@ questions:
   type: boolean
   group: "General settings"
 - variable: skipLinkerd
-  label: Skip Linkerd installation
+  label: Use existing Linkerd Installation
   required: false
   type: boolean
   group: "General settings"
 - variable: skipTraefik
-  label: Skip Traefik installation
+  label: Use existing ingress controller
```

Comment:
Not sure if we can run on any other ingress controller yet. We need to try that. We have some Traefik specific annotations to enforce tls but they shouldn't be necessary.

Reply:
We should probably make those configurable through the chart as well?

Reply:
Resolving for same reason as above. https://github.com/epinio/helm-charts/issues/36